admin/includes/languages/english/orders.php

```diff
@@ -124,7 +105,6 @@
 
 define('TABLE_HEADING_CUSTOMER_COMMENTS', 'Customer<br />Comments');
 define('TEXT_COMMENTS_YES', 'Customer Comments - YES');
-define('TEXT_COMMENTS_NO', 'Customer Comments - NO');
```

Comment:
Given this is used in relation to a switch on the page, probably should keep it for ease of embracing if one wanted to use the No option instead of Yes.

Reply:
Roger that.